Readme.md

```diff
@@ -56,6 +56,63 @@ Similarly, you can execute `npm run dotnet-postclean --if-present` via the `dotn
 To install *pnpm* globally run this command: `npm install pnpm -g`. Once it's complete, the module will automatically use that to restore packages.
 
 
+## Global NPM vs Userspace NPM via Node Version Manager on Linux
+
+If you installed NPM from your package manager, you will likely suffer an `EACCES` error when you try to install a package globally. The [recommended solution](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally#reinstall-npm-with-a-node-version-manager) is to install NPM via the Node Version Manager (NVM). This is what you should do:
+
+Before we start, ensure your directories are set up:
+```shell
+[ -z "$XDG_CONFIG_HOME" ] && echo 'export XDG_CONFIG_HOME="$HOME/.config"' >> ~/.bashrc && source ~/.bashrc
+mkdir -p "$XDG_CONFIG_HOME"
+mkdir -p "$HOME/.local/bin"
+```
+
+Also if you have installed npm globally via your package manager, uninstall it to avoid future confusion.
+
+Next we install NVM and Node.js in userspace.
+1. Install NVM for your user: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash`
+2. Type `nvm`. 
+    - If you get _nvm: command not found_ error, reload your shell profile first (`source ~/.bashrc`) and try again.
+3. Install the latest Node.js with `nvm install node`.
+    - If you are going to use Gulp and you have problems with Node.js 16.x (see [here](https://github.com/Lombiq/Orchard-Vue.js#prerequisites)), try downgrading to 14.x and make it the default: `nvm install 14.7.0 && nvm alias default 14.7.0`.
+
+This is good enough for launching new apps, but for example MSBuild doesn't use a login shell.
+If you have a desktop environment (through a display manager) see if the following works on your system. If yes, you can skip the rest of this section.
+1. Open the _~/.bashrc_ in a text editor.
+2. Open or create the _~/.xsessionrc_ file in a text editor. This is the autorun file for your dektop login session.
+3. Copy the new lines at the bottom of _~/.bashrc_ created by the NVM installer and append them to the end of the _~/.xsessionrc_.
+4. Save everything, log out and log back it.
+If everything went well you can now successfully build your Lombiq.Npm.Targets-using project as your IDE will already be in an NVM enabled environment when you start it up. 
```

Comment:
```suggestion
If everything went well you can now successfully build your `Lombiq.Npm.Targets`-using project as your IDE will already be in an NVM enabled environment when you start it up. 
```

Reply:
Aren't projects better wrapped with `_`? Just like paths, they aren't code.

Reply:
Zoltán has taught me to discern between the name in code, i.e. `Lombiq.Npm.Targets`, and the project name, i.e. Lombiq NPM Targets. But even in the latter case, it's not a path.

Reply:
I meant that project names are similar to paths in that both are technical strings that aren't code.

Reply:
"Lombiq.Npm.Targets" is the technical project name, i.e. a technical identifier and thus should be written `Lombiq.Npm.Targets`. (Conversely, "Lombiq NPM MSBuild Targets" isn't, and needn't any special treatment.) If you're referring to the folder "Lombiq.Npm.Targets" then it would be _Lombiq.Npm.Targets_.

Reply:
okay then